### DISCUSSION TITLE
8080 interface and st7789 driver

### DISCUSSION THREAD

**Discussion by Pas92** [microsoft/devicescript#587] [Q&A] [answered]
Hello everyone. I want to try Devicescript and I've bought Lilygo t-display-s3 board for it https://www.lilygo.cc/products/t-display-s3  

This board has a 1.9-inch display with st7789 driver, which is connected by 8080 interface. As I understand, ST7789 Devicescript driver uses only SPI interface. My question is how to work with display in this case?

**Comment by mmoskal** [comment, marked as answer]
Unfortunately the parallel interface is currently unsupported. Maybe it's possible to switch display into spi mode with some pin settings?

**Reply by Pas92** [reply]
Unfortunately not :( IM pins directly connected to GND. It means the 8080 interface only supported for this board. My fault. I had to read DS docs carefully before purchasing

**Reply by pelikhan** [reply]
Ouch, we'll gladly add a caution note about this detail. Do you recall which documentation page you looked at?

**Reply by pelikhan** [reply]
I've added a note in docs https://github.com/microsoft/devicescript/commit/39215eb147b75ec73b93fb0f5fa05b4c715c65b7

**Reply by Pas92** [reply]
It would be great if you also add a caution here https://microsoft.github.io/devicescript/developer/graphics  
Thanks!

**Reply by pelikhan** [reply]
added!
